external/runtime/runtime.depproj

```diff
@@ -25,16 +25,16 @@
 
   <ItemGroup Condition="'$(TargetGroup)'!='uapaot'">
     <PackageReference Include="Microsoft.NETCore.Platforms">
-      <Version>2.0.0-$(CoreFxExpectedPrerelease)</Version>
+      <Version>$(PlatformPackageVersion)</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.NETCore.Runtime.CoreCLR">
-      <Version>2.0.0-$(CoreClrExpectedPrerelease)</Version>
+      <Version>$(CoreClrPackageVersion)</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.NETCore.TestHost">
-      <Version>2.0.0-$(CoreClrExpectedPrerelease)</Version>
+      <Version>$(CoreClrPackageVersion)</Version>
     </PackageReference>
     <PackageReference Include="runtime.native.System.Data.SqlClient.sni">
-      <Version>4.4.0-$(CoreFxExpectedPrerelease)</Version>
+      <Version>$(PackageVersion)-$(CoreFxExpectedPrerelease)</Version>
```

Comment:
Yes I started to do that but then I noticed we have multiple sni versions we care about one is this one and one is the RID specific packages so instead of adding yet another SNIVersion into dependencies.props I went this route. However I don't really care for this approach much either so we should consider changing that later. For now this part of the PR is unblocked, just need to figure out the test failures now.